src/app/crypto.py

```
@@ -1,31 +1,63 @@
 """Symmetric encryption helpers for storing OAuth secrets."""
 
 from __future__ import annotations
+import os
```

Flake8 (7.3.0)

[error] 'os' imported but unused

(F401)